community/governance/Versioning.md

```diff
@@ -0,0 +1,98 @@
+---
+id: Versioning
+title: Versioning and Sundown Policy
+---
+
+## Versioning
+
+All components of the tremor project, and all repositories follow the
+[Semantic Versioning](https://semver.org/) model.
+
+As tremor is an early stage and continuously evolving software project
+we currently have a `MAJOR` version number of `0` ( zero ).
+
+## Binary Compatibility
+
+However, we aim to make each `MINOR` release binary compatible with
+previous `MINOR` releases and we publish breaking changes in our
+documentation and in the project change log.
+
+As tremor is used continuously in production for a growing set of use
+cases - it is important to the project and to our users that binary
+compatibility is maintained despite the simultaneous need to evolve
+the project to meet emerging needs.
+
+## Deprecation Process
+
+From time to time, we mark features of the project ( such as our use of YAML
+for configuration ) as deprecated with the eventual aim of replacing deprecated
+components with replacements that better meet the needs of the tremor community.
+
+A significant change to the project, may deprecate existing functionality. When it
+does:
+* The new facilities go through an open design and review based on our [RFC Process](https://www.tremor.rs/rfc/index)
```

Comment:
can this be a local link?

Reply:
We hit snags with local links in the navigation bar - avoiding for now until we know docusaurus a little more and understand when to use local links - and what issues to avoid!